invenio_exporter/config.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# invenio-exporter is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio module that adds Exporter programmatic API."""
+
+# TODO: This is an example file. Remove it if your package does not use any
+# extra configuration variables.
+
+INVENIO_EXPORTER_DEFAULT_VALUE = 'foobar'
+"""Default value for the application."""
+
+INVENIO_EXPORTER_BASE_TEMPLATE = 'invenio_exporter/base.html'
+"""Default base template for the demo page."""
```

Comment:
minor: IIRC this module will not have templates (UI), no? If so you can remove this config vars + the `/templates` difectory (see a few files below).